### ISSUE TITLE
Question about translating Android Studio projects

### ISSUE THREAD

**Issue by ikarus23** [nijel/weblate#696]
Hi,
first of all, thanks for this awesome project. I would really like to use it for my Android app. But I'm not sure how to set up the sub project to work with Android's string resources. What I did:
* Monolingual base language file: <app name>/app/src/main/res/values/strings.xml
* File mask: <app name>/app/src/main/res/values*/strings.xml

But when I'm looking at the overview, the translation box is empty. What is it I'm doing wrong? Or is it a bug? I'm using weblate 2.2

**Comment by nijel** [comment]
Did you get any error while importing? The mask should be more likely ```app/src/main/res/values-*/strings.xml```, but that should hurt only language detection and not much else...

**Comment by ikarus23** [comment]
Hey Michal,
I get `Format of translation base file could not be recognized.` as error when importing. Therefore I changed file format to `Android String Resource` manually. But your hint regarding the mask was pure gold ;). I deleted the subproject and all generated languages and re-added it with a mask containing the "-" at the end of "values". (It was really stupid of me to miss this in the first place...)
As far as I can see, everything works perfect now. Thank you for your help!
